test/dummy/config/initializers/mountain_view.rb

```diff
@@ -1,3 +1,6 @@
 MountainView.configure do |config|
   config.included_stylesheets = ["global"]
 end
+
+# include component classes
+Dir.glob('app/components/**/*.rb') { |f| require_relative "../../#{f}" }
```

Comment:
I think it'd be possible to add a line to the `MountainView::Engine` to add the components path as a Rails `autoload_path`, so users don't have to add anything to their initializers.

Something in the lines of the following should work: (haven't tested it)
```ruby
# in lib/mountain_view/engine.rb
initializer "mountain_view.append_component_paths" do |app|
  app.autoload_paths += [MountainView.configuration.components_path]
end
```
It'd probably have to include the sub-directories, so Rails doesn't fail to load the files because there's no namespace.

What do you think?